lib/screenplay/alerts/alert.ex

```diff
@@ -61,7 +61,7 @@ defmodule Screenplay.Alerts.Alert do
         }
 
   def fetch(get_json_fn \\ &V3Api.get_json/2) do
-    case get_json_fn.("alerts", %{
+    case get_json_fn.("/alerts", %{
```

Comment:
question: Do all other `get_json` calls need to include this prepended slash?

Reply:
@cmaddox5 yes. I believe I updated them all. [There aren't many](https://github.com/search?q=repo%3Ambta%2Fscreenplay%20V3Api.get_json&type=code).

This is something the `URI.append_path/2` function is picky about:

```elixir
iex(1)> base_url = "https://v3-api.mbta.com/"
"https://v3-api.mbta.com/"
iex(2)> url = base_url |> URI.parse() |> URI.append_path("alerts")
** (ArgumentError) path must start with "/", got: "alerts"
    (elixir 1.15.7) lib/uri.ex:1014: URI.append_path/2
    iex:2: (file)
```

Reply:
O woops, this is Screenplay. For whatever reason I was thinking this is Screens 😅 Sorry about that.